web/src/main/java/com/epam/esm/web/natsController/tagController/TagCreateNatsController.kt

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.web.natsController.tagController
+
+import com.epam.esm.NatsSubject
+import com.epam.esm.TagOuterClass.CreateTagRequest
+import com.epam.esm.TagOuterClass.CreateTagResponse
+import com.epam.esm.model.entity.Tag
+import com.epam.esm.service.TagService
+import com.epam.esm.web.converter.TagConverter
+import com.epam.esm.web.natsController.NatsController
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+
+@Component
+class TagCreateNatsController(
+    private val tagConverter: TagConverter,
+    private val service: TagService,
+    override val connection: Connection
+) : NatsController<CreateTagRequest, CreateTagResponse> {
```

Comment:
```suggestion
): NatsController<CreateTagRequest, CreateTagResponse> {
```
please fix in all places)